pkg/theme/style.go

```diff
@@ -6,20 +6,7 @@ import (
 	"github.com/jesseduffield/lazygit/pkg/utils"
 )
 
-var colorMap = map[string]struct {
-	foreground style.TextStyle
-	background style.TextStyle
-}{
-	"default": {style.FgWhite, style.BgBlack},
-	"black":   {style.FgBlack, style.BgBlack},
-	"red":     {style.FgRed, style.BgRed},
-	"green":   {style.FgGreen, style.BgGreen},
-	"yellow":  {style.FgYellow, style.BgYellow},
-	"blue":    {style.FgBlue, style.BgBlue},
-	"magenta": {style.FgMagenta, style.BgMagenta},
-	"cyan":    {style.FgCyan, style.BgCyan},
-	"white":   {style.FgWhite, style.BgWhite},
-}
+var colorMap = style.ColorMap
```

Comment:
This line can also be removed and somewhere below we can directly use `style.ColorMap`